app/main.py

```diff
@@ -1,2 +1,74 @@
+from typing import Any, Union
+
+from app.point import Point
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.size = 0
+        self.hash_table = [[None]] * 8
+
+    def resize(self) -> list:
+        self.hash_table += [[None]] * len(self.hash_table)
+        for el in self.hash_table:
+            if el[0] is not None:
+                index = hash(el[0]) % len(self.hash_table)
+                self.hash_table.insert(index, self.hash_table.pop(
+                    self.hash_table.index(el)))
+                self.__setitem__(el[0], el[1])
+        return self.hash_table
```

Comment:
u can just copy hash_table to the other list and then clear a self.hash_table variable and * 2(cause every resize list should become bigger in 2 times), then iter through the list copied from hash table an if elem is not none call __setitem__ method